Frends.LDAP.SearchObjects/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## [2.2.0] - 2025-03-17
```

Comment:
While we are here and editing this, can we also change the result's AttributeSet to a Dictionary<string, dynamic>? This would def make this a breaking change, but would make this task's result much more usable.

Reply:
Sure